labs/src/main/scala/org/scalalabs/basic/lab01/OOExercise.scala

```diff
@@ -40,6 +40,28 @@ import scala.language.implicitConversions
  *   of type [[org.scalalabs.basic.lab01.CurrencyConverter]]
  * - Use the implicit CurrencyConverter to do the conversion. 
  */
-class Euro {
+class Euro(val euro: Int, val cents: Int) extends Currency("EUR") {
+  def this(euro: Int) = this(euro, 0)
```

Comment:
Since the difference in constructors is just a default value, you could also accomplish this by `class Euro(val euro: Int, val cents: Int = 0)`.